### ISSUE TITLE
Installation by bower not work

### ISSUE THREAD

**Issue by krajcikondra** [nittro/nittro#11]
I tryed to install nitro by **bower install nittro --save**. Package was download with folders (Ajax, Core, DateTime,. ..) but folders were empty.

**Comment by jahudka** [comment]
Hi, I'm sorry about that, but starting with Nittro 2.0 which got quietly released yesterday night we're using bower and npm a bit differently then we used to. There are two new packages in bower called `nittro-essentials` and `nittro-full` and these contain prebuilt nittro packages suitable for use in situations where you don't need a custom build; they are the same thing as the ZIP files you can download from https://nittro.org/download. The source components are available via NPM, along with Gulp and Grunt plugins which you can use to create a customised Nittro build.

The reason is that in order to use automated build tools like Gulp or Grunt you need to have Node and NPM anyway, so there's no reason for Nittro to require you to use an additional package manager on top of NPM if you're building a custom package.

**TL;DR** - to make your problem go away, simply switch from Bower to NPM for the `nittro-*` packages (`npm install --save nittro`).
